### ISSUE TITLE
User specific recent activity feed

### ISSUE THREAD

**Issue by jhrdt** [BookStackApp/BookStack#2993]
**Describe the feature you'd like**

* A user should be able to observe shelves/books/pages (like marking a page as favorite)
  * The page level would be the most relevant of those three 
* A user should have the option "Activity Feed" in his user menu
  * This could also be a tile in the "My Profile" view
* A counter should appear inside the profile logo. The counter shows the number of new (not seen) activities.
* A activity is marked as "seen", when the use visits the feed page.
* Update or create events should be listed in the feed. Each item should link to the affected shelve/page/book
  * De facto this is only a user defined subset of the recent activity feed

**Describe the benefits this feature would bring to BookStack users**

This would be a way for a user to keep track of recent changes in a self-defined way.
A user gets all information about the relevant changes, without the need for scanning through the recent activity view. In a very active BookStack app, the relevant changes are filled very quickly.

Use Case:
This feature is a benefit, when there are different working groups that overlap only in a subset of pages that are relevant to them.

**Comment by jhrdt** [comment]
Hi @ssddanbrown ,

I totally forgot about comment. This watch feature should also include notifications about comments. I think comments are great, but they are hard to follow, this would improve their use.

**Comment by hristijan5** [comment]
I would love the idea of being notified of a change in a page (maybe a dedicated area in the profile that will list the notifications?).
BTW Great job on the overall BookStack app.

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/2993
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
